ports/zephyr/boards/xiao_ble_nrf52840_sense/manifest.py

```diff
@@ -0,0 +1,8 @@
+# freeze("../modules")
+include("$(MPY_DIR)/extmod/asyncio")
+
+# Require a micropython-lib module.
```

Comment:
This comment is not really correct, it was copied from the example manifest.

You could remove it, or change it to something like "Require some useful micropython-lib modules."